### PR TITLE
I added a feature

### PR DESCRIPTION
**Pull request by Eilux**
For use in a mod I was making I added a shouldOverwrite() method that will make the pack invisible and apply it over the vanilla game. I also updated the gradle.properties so it will be in 1.16.3.

### CODE REVIEW

File: src/main/java/com/swordglowsblue/artifice/api/ArtificeResourcePack.java [hunk 1]
```diff
@@ -63,6 +63,8 @@
      */
     boolean isVisible();
 
+    boolean isShouldReplace();
+
```
Comment (natanfudge) [inline src/main/java/com/swordglowsblue/artifice/api/ArtificeResourcePack.java:67]: Explain what this does

**Review by natanfudge — CHANGES_REQUESTED** [review]
Seems reasonable

File: src/main/java/com/swordglowsblue/artifice/api/ArtificeResourcePack.java (hunk 1 above)
Comment (Eilux) [reply src/main/java/com/swordglowsblue/artifice/api/ArtificeResourcePack.java:67]: It returns a Boolean value that says if it should overwrite Or not (I realize now that I used replace and overwrite interchangeably) mostly for testing purposes.

Comment (Eilux) [reply src/main/java/com/swordglowsblue/artifice/api/ArtificeResourcePack.java:67]: I have updated the naming to be more consistant

**Pull request merged**